docs/exchange-operators/faq.mdx

```diff
@@ -54,7 +54,7 @@ The max mempool size is 3,000. After it hits that size, transactions with the lo
 
 Set your fee to an amount higher than 0.001 MINA, the current average fee for transactions in the pool.
 
-You can view the fees for pending transactions and adjust your fees accordingly: [https://minaexplorer.com/pending-transactions](https://minaexplorer.com/pending-transactions).
+You can view the fees for pending transactions and adjust your fees accordingly: [https://minascan.io/devnet/txs](https://minascan.io/devnet/txs).
```

Comment:
Should it be mainnet instead of devnet?

Reply:
should probably be - https://minascan.io/mainnet/txs/pending-txs - I'll update

Reply:
I've updated + made the rosetta section callapsible so it matches the other doc sections.